### PR TITLE
Integration with ODIM's services part #8

### PR DESCRIPTION
**Pull request by MaciejMis**
Changes:
- Updated configs for Device Manager and ODIM,
- Upgraded Go version to 1.17

### CODE REVIEW

File: lib-utilities/config/odimra_config.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+	"LocalhostFQDN": "deviceManager",
+	"SearchAndFilterSchemaPath": "/etc/deviceManager/configs/schema.json",
+	"RegistryStorePath": "/etc/deviceManager/registryStore/",
 	"KeyCertConf": {
-		"RootCACertificatePath": "",
-		"RPCPrivateKeyPath": "",
-		"RPCCertificatePath": "",
-		"RSAPublicKeyPath": "",
-		"RSAPrivateKeyPath": ""
+		"RootCACertificatePath": "/etc/deviceManager/certs/rootCA.crt",
```
Comment (ehaligow) [inline lib-utilities/config/odimra_config.json:7]: Path for certs and keys is different than in /src/config/config.yml, are they different certs? If not, can we set the same path?

File: lib-utilities/config/odimra_config.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
 {
-	"RootServiceUUID": "",
-	"LocalhostFQDN": "",
-	"SearchAndFilterSchemaPath": "",
-	"RegistryStorePath": "",
+	"RootServiceUUID": "ac2f7880-9d8a-11ed-a8fc-0242ac120002",
+	"LocalhostFQDN": "deviceManager",
+	"SearchAndFilterSchemaPath": "/etc/deviceManager/configs/schema.json",
```
Comment (ehaligow) [inline lib-utilities/config/odimra_config.json:4]: Can't we use schema.json file located in /lib-utilities/config?

Comment (MaciejMis) [reply lib-utilities/config/odimra_config.json:4]: Configs and certs should be moved to /etc/deviceManager/ directory.

File: lib-utilities/config/odimra_config.json (hunk 1 above)
Comment (MaciejMis) [reply lib-utilities/config/odimra_config.json:7]: Please note that `config.yml` was updated too to match paths between two configs.

**Pull request merged**